Doc/using/cmdline.rst

```diff
@@ -539,8 +539,16 @@ Miscellaneous options
      application.  Typical usage is ``python3 -X importtime -c 'import
      asyncio'``.  See also :envvar:`PYTHONPROFILEIMPORTTIME`.
 
+     ``-X importtime=2`` enables additional output that indicates when an
```

Comment:
Please update also the `--help-xoptions` output. Do not add much, just make clear that it can take argument (`-X importtime[=2]`) and what it means.

I wonder if we should also support `=0` to disable the output. But this can wait for another issue, many other options could be harmonized.

Reply:
Deferring `=0` to another issue/PR as suggested.